src/gb/gbSGB.cpp

```diff
@@ -67,7 +67,7 @@ void gbSgbReset()
     gbSgbNextController = 0x0f;
     gbSgbReadingController = 0;
 
-    memset(gbSgbSCPPalette, 0, 512 * 4);
+    //memset(gbSgbSCPPalette, 0, 512 * 4 * sizeof(uint16_t));
```

Comment:
typo //

Reply:
I think he did this because it's a global and globals are zero-initialized implicitly, in which case just delete it.

Reply:
Yes, exactly this. `gbSgbSCPPalette` is a struct and its values are zero-initialized.

Reply:
its part of gbSgbReset which is called everytime you reset the game...

Reply:
Ok, I understand. I will undo this on next commit.